src/nodes/ModifierDefinition.js

```diff
@@ -38,16 +32,22 @@ const override = (node, path, print) => {
 
 const body = (node, path, print) => {
   if (!node.body) return ';';
-  if (node.isVirtual) return group([' ', path.call(print, 'body')]);
-  return [' ', path.call(print, 'body')];
+  if (node.isVirtual) return group(path.call(print, 'body'));
+  return [path.call(print, 'body')];
 };
 
 export const ModifierDefinition = {
   print: ({ node, path, print }) => [
     'modifier ',
     node.name,
     modifierParameters(node, path, print),
-    group(indent([virtual(node), override(node, path, print)])),
+    group(
```

Comment:
ModifierDefinition also did not follow the same rules we discussed for FunctionDefinitions.